lib/SymbolGraphGen/SymbolGraph.cpp

```diff
@@ -232,6 +232,18 @@ void SymbolGraph::recordEdge(Symbol Source,
 
 void SymbolGraph::recordMemberRelationship(Symbol S) {
   const auto *DC = S.getLocalSymbolDecl()->getDeclContext();
+  const ValueDecl *ParentDecl = DC->getSelfNominalTypeDecl();
```

Comment:
Do we also need to support the other kinds of relationships like `inheritsFrom`, `requirementOf`, etc.?

Reply:
I handled the other kinds of relationship that are necessary for type declarations in https://github.com/swiftlang/swift/pull/78959/commits/b7079f4eca002113e7c64c0e61748be79a4c75a7 - this includes `inheritsFrom` and `conformsTo`, since `requirementOf` only applies to method declarations on protocols. Any other relationship kind that applies to the synthesized children will happen as these symbols are processed.